content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -282,22 +286,26 @@ kubeadm 不支持将没有 `--control-plane-endpoint` 参数的单个控制平
 <!--
 ### More information
 -->
-### 更多信息
+### 更多信息 {#more-information}
 
 <!--
 For more information about `kubeadm init` arguments, see the [kubeadm reference guide](/docs/reference/setup-tools/kubeadm/).
 -->
 有关 `kubeadm init` 参数的更多信息，请参见 [kubeadm 参考指南](/zh/docs/reference/setup-tools/kubeadm/)。
 
 <!--
-To configure `kubeadm init` with a configuration file see [Using kubeadm init with a configuration file](/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file).
+To configure `kubeadm init` with a configuration file see
+[Using kubeadm init with a configuration file](/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file).
 -->
 要使用配置文件配置 `kubeadm init` 命令，请参见[带配置文件使用 kubeadm init](/zh/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file)。
```

Comment:
```suggestion
要使用配置文件配置 `kubeadm init` 命令，请参见
[带配置文件使用 kubeadm init](/zh/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file)。
```

Reply:
Okay, but to prevent the space appears before link, I will split the line at this point.

```suggestion
要使用配置文件配置 `kubeadm init` 命令，
请参见[带配置文件使用 kubeadm init](/zh/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file)。
```